internal/coreinternal/aggregateutil/aggregate.go

```diff
@@ -75,13 +77,13 @@ func GroupDataPoints(metric pmetric.Metric, ag *AggGroups) {
 func MergeDataPoints(to pmetric.Metric, aggType AggregationType, ag AggGroups) {
 	switch to.Type() {
 	case pmetric.MetricTypeGauge:
-		mergeNumberDataPoints(ag.gauge, aggType, to.Gauge().DataPoints())
+		mergeNumberDataPoints(pmetrictest.OrderMapByKey(ag.gauge), aggType, to.Gauge().DataPoints())
```

Comment:
Could we either duplicate this functionality or include it somewhere as a non-testing library? It feels odd to rely on a test package like this.

Reply:
I moved it to `internal/common/maps`, any better suggestions?